haystack/database/elasticsearch.py

```diff
@@ -89,7 +91,7 @@ def get_document_by_name(self, name):
         return document
 
     def get_document_ids_by_tags(self, tags):
-        term_queries = [{"terms": {key: value}} for key, value in tags.items()]
+        term_queries = [{"term": {key: value}} for key, value in tags.items()]
```

Comment:
I guess you tried `finder.get_answers(..., filters={"tag": "some-value"})` which caused the error? 
I just saw that our docs are outdated here. Sorry for that. It should be  `finder.get_answers(..., filters={"tag": ["some-value"]})`. We switched to the terms query since it has the advantage of allowing multiple filter values for the same field and uses some additional caching (https://www.elastic.co/guide/en/elasticsearch/reference/7.6/query-dsl-terms-query.html). Can you try if this solves your previous problem?

Reply:
Ah, yes, that does the trick! Works fine. Thanks!

Reply:
Perfect :)